tools/autograd/derivatives.yaml

```diff
@@ -470,6 +470,16 @@
 - name: floor(Tensor self) -> Tensor
   self: zeros_like(grad)
 
+- name: float_power.Tensor_Scalar(Tensor self, Scalar exponent) -> Tensor
```

Comment:
With the `Math:` dispatch and calling `pow` I don't think this will need its own derivatives.yaml entries.

Reply:
Addressed. But there is another problem as mentioned above.